### DISCUSSION TITLE
Unable to update via AltStore

### DISCUSSION THREAD

**Discussion by crowbardispensingcore** [qnblackcat/uYouPlus#546] [Q&A]
I've been having this issue for so long I don't even remember if I ever updated uYouPlus successfully via AltStore at all. Every time I update, it will stuck at the last 1 percent forever, and I always have to delete uYouPlus and install the new version to make it work, which means every time I need to set every setting again... It doesn't matter if I'm connected to AltStore via WiFi or cable, or if I choose Keep Extensions Or Not before installation. Does anyone else has the same issue?

**Comment by qnblackcat** [comment]
https://github.com/qnblackcat/uYouPlus/wiki/FAQ

**Reply by WoodpeckerBaby** [reply]
Why this app always fails to install in particular? ![image](https://user-images.githubusercontent.com/14303318/198732645-0aa7b4bf-f152-4b98-bbdc-a89c522b3a15.jpeg)

It's so frustrating I stopped updating it.

![image](https://user-images.githubusercontent.com/14303318/198732871-c2c27d09-68d6-4a33-ba0c-5955085e4a49.jpeg)

**Reply by crowbardispensingcore** [reply]
> https://github.com/qnblackcat/uYouPlus/wiki/FAQ

Didn't find anything in the faq that addresses the issue. It would be nice to at least know if it's supposed to work or not.
